report.go

```diff
@@ -55,11 +59,31 @@ type TestArtifact struct {
 	EventsHistogram       bool      `bson:"events_histogram,omitempty" json:"events_histogram,omitempty" yaml:"events_histogram,omitempty"`
 	EventsIntervalSummary bool      `bson:"events_interval_summary,omitempty" json:"events_interval_summary,omitempty" yaml:"events_interval_summary,omitempty"`
 	EventsCollapsed       bool      `bson:"events_collapsed,omitempty" json:"events_collapsed,omitempty" yaml:"events_collapsed,omitempty"`
+	ConvertGzip           bool      `bson:"convert_gzip,omitempty" json:"convert_gzip,omitempty" yaml:"convert_gzip,omitempty"`
+	ConvertBSON2FTDC      bool      `bson:"convert_bson_to_ftdc,omitempty" json:"convert_bson_to_ftdc,omitempty" yaml:"convert_bson_to_ftdc,omitempty"`
+	ConvertCSV2FTDC       bool      `bson:"convert_csv_to_ftdc" json:"convert_csv_to_ftdc" yaml:"convert_csv_to_ftdc"`
 }
 
 func (a *TestArtifact) Validate() error {
 	catcher := grip.NewBasicCatcher()
 
+	if a.ConvertGzip {
+		a.DataGzipped = true
+	}
+
+	if a.ConvertCSV2FTDC {
+		a.PayloadFTDC = true
+
+	}
+	if a.ConvertBSON2FTDC {
+		a.PayloadBSON = false
```

Comment:
Can this be set true anywhere else? Otherwise false would be the default and this isn't needed.

Reply:
presumably by users.